rotkehlchen/tests/api/test_metadata.py

```diff
@@ -11,10 +11,9 @@
     from rotkehlchen.api.server import APIServer
 
 
-@pytest.mark.vcr()
+@pytest.mark.skip('TODO: @yabirgb. Re-record this test when we merge the v2 schema')
```

Comment:
what's this?

Reply:
The tests makes querests to the data repo to fetch and process the actual file but since the file doesn't exist because we haven't merged the changes there yet the test can't be vcre'd